data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,172 @@
+			choice
+				`	"OK"`
+					accept
+				`	"Sorry, not interested."`
+					decline
+	on complete
+		payment 50000
+		conversation
+			`The <ship> has not even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple spots down to stare up at the gills of an Albatross. A few moments later, Taely drops down to the ground beside her. As you approach you can see Ruach has a holo of the spectral ship floating in the air between them. "It is lacking the confirmation codes or identification info, but I am confident that this is it." Ruach says as you approach.`
+			`	"Sorry for leaving you behind, this is just such an exciting discovery. This ship you have found matches the profile of the Shattered Light - a large city ship that was part of the exodus but never arrived on this side of the wormhole. There are so many questions here! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`
+			
+
+
+mission "Remnant: Shattered Light 3"
+	invisible
+	name "Ghost Touch"
+	description "Help the Remnant experiment on the Shattered Light."
+	blocked "As you approach the starport your commlink chimes with a message from Ruach requesting that you make space for 13 people as well as 19 cargo."
+	cargo "equipment" 19
+	passengers 13
+	source "Viminal"
+	destination "Far Monad"
+	to offer
+		has "Remnant: Shattered Light 2: done"
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system Lucina
+		ship "Dromedary Specter" "???"
+		dialog `It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`
+	on offer
+		conversation
+			`The group of people standing around a hologram over on one side of the cafeteria. Ruach gestures for you to join them, and introduces you to each of the people in the group. "I would just like to start by offering our thanks for your assistance, Captain <first>. You have sharp eyes to pick this up as early as you did. Other scouts reported it shortly thereafter, but you spotted it first." She gestures to the hologram of the ship floating in front of you.`
+			`	"We have confirmed that this matches all available records of the Shattered Light, although it appears to have suffered considerable damage. One of the many mysteries here is what has happened to it, and what we should do about it."`
+			`	Another Remnant steps forward and the hologram changes to show a scene from a cave with a large object. Other Remnant appear to aim some equipment at it, and after a moment or two, the object expands to several times its original size. "These are recordings from our explorations of the ruins, demonstrating the decompression of a Maudrakoi artifact. While not identical, these artifacts generate similar readings to those recorded during your scan of the Shattered Light.`
```

Comment:
What's a Maudrakoi? Or explain it's origin (ruins found on Viminal for example).

Reply:
The name of the aliens whose ruins abound on the Remnant worlds. I'll add some conversation options to let the player ask about it.